app/server/tests/handleError.test.ts

```diff
@@ -1,7 +1,6 @@
-/* eslint-disable import/first */
 // mock json response before import errors
-const mockJsonResponseError = jest.fn();
-jest.mock("../src/jsonResponse", () => { return { jsonResponseError: mockJsonResponseError }; });
+const { mockJsonResponseError } = vi.hoisted(() => ({ mockJsonResponseError: vi.fn() }));
+vi.mock("../src/jsonResponse", () => { return { jsonResponseError: mockJsonResponseError }; });
```

Comment:
`mockJsonResponseError` has to be hoisted as it is used in the `vi.mock` and in the tests